### DISCUSSION TITLE
Parabolic SAR returning different values than TradingView

### DISCUSSION THREAD

**Discussion by paramjit2k** [DaveSkender/Stock.Indicators#909] [Q&A] [answered]
Hi

Parabolic SAR is returning different trend values in comparison to TradingView. All the settings are the same.

ParabolicSARresults = HeikenAshiResults.GetParabolicSar(0.02, 0.2, 0.02);

**Comment by DaveSkender** [comment, marked as answer]
> Hi
> 
> Parabolic SAR is returning different trend values in comparison to TradingView. All the settings are the same.
> 
> ParabolicSARresults = HeikenAshiResults.GetParabolicSar(0.02, 0.2, 0.02);

Let's continue the conversation in your post on [#245](https://github.com/DaveSkender/Stock.Indicators/discussions/245#discussioncomment-3821061), but also review #1031 and #801
